OmniKit/MessageTransport/Message.swift

```diff
@@ -16,6 +16,23 @@ public enum MessageError: Error {
     case validationFailed(description: String)
 }
 
+extension MessageError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .notEnoughData:
+            return LocalizedString("Not enough data", comment: "Description for MessageError notEnoughData")
+        case .invalidCrc:
+            return LocalizedString("Invalid CRC", comment: "Description for MessageError invalidCrc")
+        case .parsingError:
+            return LocalizedString("Parsing Error: ", comment: "Description for MessageError parsingError")
+        case .unknownValue(let value, let typeDescription):
+            return String(format: LocalizedString("Unknown Value (%1$@) for type %1$@", comment: "Format string for description of MessageError unknownValue. (1: value) (2: Type)"), String(describing: value), typeDescription)
```

Comment:
Is this `Unknown Value (%1$@) for type %1$@` correct?

Reply:
Ah, good catch. Thanks.